cernopendata/modules/theme/static/assets/ispy/ispy.css

```diff
@@ -0,0 +1,12 @@
+.ispy-help-img {
+    width: 100%;
+    height: auto;
+}
+
+.ispy-help-item {
+    padding: 5px !important;
+}
+
+.ispy-help-row {
+    margin: 0 !important;
+}
```

Comment:
[Using `!important` is a bad practice](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity#the_!important_exception) as it breaks the natural cascading and makes debugging harder.

This can be avoided with something like:

```css
.bootstrap-ispy .ispy-help-item {
    padding: 5px;
}

.bootstrap-ispy .ispy-help-row {
    margin: 0;
}
```

Tested and working properly.

Reply:
Interesting! I had just rebuilt afresh this PR to confirm/infirm the problem, and I do get the same non-arranged setup described in the issue. Seeing this with Firefox, Chromium on Fedora 33. What could be the reason for the rendering difference on our two platfroms? Perhaps the font setup or some such? BTW, do you see the original problem on `master` branch e.g. if you check [on DEV](https://opendata-dev.cern.ch/visualise/events/cms)?

Reply:
Yes, I see the problem on DEV. FF on macOS 11.2 Big Sur.

Reply:
OK, here is the problem. Using this PR, I'm seeing the following:

![out](https://user-images.githubusercontent.com/517546/108194495-ff37d380-7116-11eb-9643-64a820568d78.png)

Note how "out." is displayed on the next line. This is not what you see in the screenshot, and this is exactly what makes the arrangement to go wild. 

Using browser inspector, I shortened the text, and the boxes became well aligned. So the workaround solution could be to simply shorten the help text and write "Zoom in/out", as follows:

![out3](https://user-images.githubusercontent.com/517546/108195044-9ac94400-7117-11eb-8bb9-c2c6e0b46d53.png)

This fixes the arrangement of the boxes on my setup as well.

Perhaps you can add this improvement to the PR alongside your `!important` changes?

(Alternatively, we could find out why the font size is not "the same" on our two platforms, but it does not seem worth it...)

Reply:
I see.. tricky indeed.. this might be related to the problem we had with mathjax and so... In any case, we should have a better grid that prevent these arrangement issues, I guess we can tackle it on a different issue together with the font size problem.

I updated the PR, could you have a look now and merge if it's ok?